contracts/mock/anchor/AnchorStrategy.sol

```diff
@@ -85,7 +85,7 @@ contract AnchorStrategy is IAnchorStrategy, BaseStrategy {
         IERC20 _ustToken,
         IERC20 _aUstToken,
         address _admin
-    ) {
+    ) BaseStrategy(_vault, address(_ustToken), _admin) {
         if (_admin == address(0)) revert StrategyAdminCannotBe0Address();
```

Comment:
I guess you can remove some of these lines?

Reply:
I wasn't concerned with making changes to anchor stuff, but I can make the changes anyway

Reply:
[fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/183/commits/35ca1a303f2f8b8df6b9c67db9b9fd793e49d8be)

Reply:
It was mostly for the tests, but I guess it's a mock contract, so it doesn't make a difference